CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 * [BUGFIX] Ingester: When using block storage, prevent any reads or writes while the ingester is stopping. This will prevent accessing TSDB blocks once they have been already closed. #4304
 * [BUGFIX] Ingester: fixed ingester stuck on start up (LEAVING ring state) when `-ingester.heartbeat-period=0` and `-ingester.unregister-on-shutdown=false`. #4366
 * [BUGFIX] Querier: After query-frontend restart, querier may have lower than configured concurrency. #4417
+* [FEATURE] Distributor: Add label `statusFamily` to metric `cortex_distributor_ingester_append_failures_total` #4441
```

Comment:
By convention, we should group all the `[FEATURE]` together; I see there is already some `[FEATURE]` tags at the top, so let's move this line there.